### PR TITLE
Bump versions to support Lumen 7.x and 8.x

### PR DESCRIPTION
**Pull request by ls-renaud-tilte**
Bump versions to support Lumen 7.x and 8.x

### CODE REVIEW

**Comment by Jalle19** [comment]
https://packagist.org/packages/nordsoftware/lumen-chained-exception-handler#2.0.0 is now tagged

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,13 @@ language: php
...
 php:
-  - 7.1
   - 7.2
   - 7.3
+  - 7.4
 
 before_script:
   - composer self-update
-  - composer install
+  - COMPOSER_MEMORY_LIMIT=-1 travis_retry composer install --prefer-dist --no-interaction
```
Comment (ls-renaud-tilte) [inline .travis.yml]: Travis failed with PHP running out of memory.

This is an easy fix (https://lorenzo.mile.si/travis-ci-and-composer-out-of-memory/739/)

File: tests/NewRelicServiceProviderTest.php [hunk 2]
```diff
@@ -46,13 +47,13 @@ public function testRegisterExceptionHandler()
 
         // Define a route that throws an exception
         $app->router->get('/', function() {
-            throw new \Exception();
+            throw new HttpException(550);
```
Comment (ls-renaud-tilte) [inline tests/NewRelicServiceProviderTest.php:50]: Exceptions are better handled with the latest version of lumen, and regular 500 errors are now spitting out HTML. 

Setting a `550` error (not existing) falls back to the regular error message `Whoops, looks like something went wrong`.

File: .travis.yml (hunk 1 above)
Comment (Jalle19) [reply .travis.yml]: I have a feeling `--prefer-dist` would have been enough?

File: tests/NewRelicServiceProviderTest.php (hunk 2 above)
Comment (Jalle19) [reply tests/NewRelicServiceProviderTest.php:50]: This is fine, it's just a test case after all.

**Comment by Jalle19** [comment]
Can you uncommit composer.lock? Seems it got accidentally added.

**Comment by ls-renaud-tilte** [comment]
> Can you uncommit composer.lock? Seems it got accidentally added.

Yep, I had issue with scrutinizer never completing and apparently committing the `composer.lock` helps getting the right version for dependency analysis, but I would have removed the lock file afterward.

**Comment by ls-renaud-tilte** [comment]
> Can you uncommit composer.lock? Seems it got accidentally added.

Any update on this one?

**Pull request merged**